src/components/Pagination/Pagination.tsx

```diff
@@ -135,7 +135,7 @@ const Pagination: React.FC<PaginationProps> = ({
         <Container>
             <ButtonsContainer>
                 {hasMultiplePageSizes && (
-                    <Box aria-label={ariaLabelSelectPageSizeContainer} position="absolute" left="0" width="4.5em">
+                    <Box aria-label={ariaLabelSelectPageSizeContainer} position="absolute" left="0">
```

Comment:
Not directly related, but I think the width here was an atempt to minimise a possible collision between the nav buttons and the and the dropdown.

<img width="735" alt="Screenshot 2024-07-23 at 16 09 51" src="https://github.com/user-attachments/assets/d7030092-7a43-4c88-95f8-852a33a04c45"> 
(taken from the running the artifact of storybook locally)

Not sure what/if something changed recently but this is not happening currently in one implementation (at least):

<img width="257" alt="Screenshot 2024-07-23 at 16 12 15" src="https://github.com/user-attachments/assets/27e668ca-deb1-4dea-855b-3dd5542d1004">

Do you think you can have a look at how we place the page size selector in the container so it has block size and make the other buttons wrap, instead of overlapping with them? 

Thanks in advance!

Reply:
Just realised that this a fix for the V1. So feel free to ignore this... Since in V2 it seems not to be an issue.

I'd make sure a patch for the 1.x would be clearly communicated in the PR description to avoid similar situation in the future 🙏🏻

Reply:
okay 👍 the pagination is not optimised for mobile at all, which is separate tech debt itself that should be resolved, for now we're just updating this for DMT (so for Wave v1).

Reply:
@arturmiglio I've commited fix for the layout as well, to not cause overlapping of elements (like in v2)